easybuild/easyblocks/p/petsc.py

```diff
@@ -224,20 +228,43 @@ def configure_step(self):
                         withdep = "--with-%s" % dep[1].lower()
                     self.cfg.update('configopts', '%s=1 %s-dir=%s' % (withdep, withdep, deproot))
 
+            # SCOTCH has to be treated separately since they add weird postfixes
+            # to library names from SCOTCH 7.0.1 or PETSc version 3.17.
+            if (LooseVersion(self.version) >= LooseVersion("3.17")):
```

Comment:
We should make sure that the condition is correct here: is this custom handling required because of SCOTCH 7.x, or PETSc >= 3.17?

My guess would be it's because of changes in SCOTCH 7.x? If so, this should become:

```python
scotch = get_software_root('SCOTCH')
scotch_ver = get_software_version('SCOTCH')
if scotch and LooseVersion(scotch_ver) >= LooseVersion('7.0.0'):
    ...

Reply:
Done